### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by dev1980**
- Create a components directory in the src folder
- Create four files in there, each one representing one React component
-App, Button, ButtonPanel, and Display components.

-

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const { value } = prop;
+
+  return (
+    <div className="display">
+      <h3>{value}</h3>
+    </div>
+  );
+};
+
+Display.defaultProps = { value: PropTypes.string.isRequired };
```
Comment (azdrenymeri) [inline src/components/Display.js]: - [x] The property `value` inside `Display.defaultProps` should be set to zero.
- [x] Also the prop type is not defined to string for `value`.
```javascript
// hint
Display.propTypes .....
```

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
#### Hello @dev1980 :wave: 
#### Great job on this milestone :+1: 
#### There is only one fix you need to make inside `Display.js`
#### Kindly fix it and add it for another review.

File: src/components/Display.js (hunk 1 above)
Comment (dev1980) [reply src/components/Display.js]: implement the requirements

**Review by MauricioRobayo — APPROVED** [review]
Thanks for fixing the issues mentioned in the previous review.

Please move on!

Status: __Approved__.

**Pull request merged**